ev3dev/motor.py

```diff
@@ -732,6 +792,46 @@ def wait_while(self, s, timeout=None):
         """
         return self.wait(lambda state: s not in state, timeout)
 
+    def rps_to_speed(self, rps):
+        """
+        Return the motor speed percentage to achieve rps (rotations-per-second)
+        """
+        assert abs(rps) <= self._max_rps, "%s maximum rotations-per-second is %s, %s was requested" % (self, self._max_rps, rps)
```

Comment:
What do you think about moving these conversions into the individual unit classes (or somewhere else)?

Reply:
sounds good...I am going to change `Motor._max_rps` to `Motor.max_rps` though since `SpeedRPS` will be accessing it to do the conversion